_posts/2017-04-02-welcome.md

```diff
@@ -5,4 +5,13 @@ author: Florian Gilcher
 description: "The Rust Community Blog is open for business!"
 ---
 
-Some blurb!
+Hi Rustaceans!
+
+The Rust Community team is happy to announce the Rust Community blog!
+
+This blog is both intended as our regular outlet for community topics, as well as a highlight community events - meetups, usergroups out of the order events.
+
+If you have a community topic you would like write about here, please [contact us](mailto:community-team@rust-lang.org), or just [open an issue](https://github.com/rust-community/blog.community.rs).
```

Comment:
We should start brainstorming what kind of standards we want for our guest posts. I'll open a ticket for that.